BUILDING.md

```diff
@@ -110,17 +125,28 @@ platforms. This is true regardless of entries in the table below.
 
 Depending on the host platform, the selection of toolchains may vary.
 
-#### Unix
-
-* GCC 4.9.4 or newer
-* Clang 3.4.2 or newer
-
-#### AIX
-* GCC 6.3 or newer
-
-#### Windows
-
-* Visual Studio 2017 with the Windows 10 SDK on a 64-bit host.
+| Operating System | Compiler Versions                                           |
+| ---------------- | ----------------------------------------------------------- |
+| Linux            | GCC >=6.3, Clang >=3.4.2                                    |
+| Windows          | Visual Studio 2017 with the Windows 10 SDK on a 64-bit host |
+| macOS            | Xcode / Apple LLVM >= 7                                     |
+
+### Official binary platforms and toolchains
```

Comment:
this isn't strictly related to users "BUILDING" Node but we've been needing to list this _somewhere_ for a long time and this seems like a good place to record everything related to our versions.